governance/program/src/state/governance.rs

```diff
@@ -205,21 +209,30 @@ impl GovernanceV2 {
         governing_token_mint: &Pubkey,
     ) -> Result<(), ProgramError> {
         // resolve_vote_threshold() asserts the vote threshold exists for the given governing_token_mint and is not disabled
-        let _ = self.resolve_vote_threshold(realm_data, governing_token_mint)?;
+        let _ = self.resolve_vote_threshold(realm_data, governing_token_mint, None)?;
```

Comment:
The `None` here looks like a potential bug because of the optional vote parameter.  It makes it seem like you're not checking that a veto is proper

Reply:
Fixed with `VoteKind`